core/tests/build_sources/templates/base.html

```diff
@@ -87,7 +87,6 @@
         isAdmin: JSON.parse('{{is_admin|js_string}}'),
         isModerator: JSON.parse('{{is_moderator|js_string}}'),
         isSuperAdmin: JSON.parse('{{is_super_admin|js_string}}'),
-        userIsLoggedIn: JSON.parse('{{user_is_logged_in|js_string}}'),
```

Comment:
You need to do the same remove in the actual [base page](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/base.html).